cards.css

```diff
@@ -0,0 +1,171 @@
+.cards-container{
+    padding: 1rem 1rem;
+}
+
+#card-img{
+    width:200px;
+    height: 200px;
+    border-radius: 5px 5px 0 0;
+}
+
+.card-box{
+    position: relative;
+    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+    transition: 0.3s;
+    height: 300px;
+    width: 200px;
+    border: 1px solid black;
+    border-radius: 5px;
+  }
+  
+.card-box:hover {
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+  }
+
+.card-box-container {
+padding: 2px;
+}
+
+.img-badge{
+    height: 20px;
+    width: 100px;
+    text-align: center;
+    color:black;
+    background-color: red;
+    border:1px solid transparent;
+}
+
+.card-img-lg{
+    width: 200px;
+    height: 200px;
+}
+
+.badge-image{
+    height: 20px;
+    width: 100px;
+    text-align: center;
+    color:black;
+    background-color: #A5B4FC;
+    position: absolute;
+    border:1px solid transparent;
+    border-bottom-left-radius: 5px;
+}
+
+.right-top{
+    right: 0px;
+    top: 0px;
+  }
+
+.card-box-overlay{
+position: relative;
+box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+transition: 0.3s;
+height: 300px;
+width: 200px;
+border: 1px solid black;
+border-radius: 5px;
+}
+
+.card-overlay {
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    height: 100%;
+    width: 100%;
+    opacity: 0;
+    transition: .5s ease;
+    background-color: #DC2626;
+  }
+  
+  .card-box-overlay:hover .card-overlay {
+    opacity: 1;
+  }
+  
+  .overlay-text {
+    color: white;
+    font-size: 20px;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    -webkit-transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%);
+    transform: translate(-50%, -50%);
+    text-align: center;
+  }
+
+
+  .card-box-shadow{
+    position: relative;
+    box-shadow: 0px 4px 18px 0 black;
+    transition: 0.3s;
+    height: 300px;
+    width: 200px;
+    border: 1px solid black;
+    border-radius: 5px;
+  }
+
+
+
+
+
+
+
+
+/* ---margin---- */
+.m-0 {
+    margin: 0;
+}
+.mr-1 {
+    margin-right: 1rem;
+}
+
+/* ----text and color----- */
+
+
+.bg-black {
+    background-color: black;
+}
+.bg-blue {
+    background-color: blue;
+}
+.bg-red {
+    background-color: red;
+}
+.bg-white {
+    background-color: white;
+}
+
+.border-round {
+    border-radius: 2rem;
+}
+
+/* ----flex----- */
+.flex {
+    display: flex;
+}
+
+.palette .color {
```

Comment:
.palette .color is this right,if it is would you brief me what is going on here?